cli/main.ml

```diff
@@ -285,7 +285,7 @@ let certfile =
   Arg.(value & opt string "" & info ["certfile"] ~doc)
 let ciphersuites =
   let doc = "Set of ciphersuites for TLS (specified in the format accepted by OpenSSL, stunnel etc.)" in
-  Arg.(value & opt string "ECDHE-RSA-AES256-GCM-SHA384" & info ["ciphersuites"] ~doc)
+  Arg.(value & opt string (failwith "cipherstring is required") & info ["ciphersuites"] ~doc) (* cli is only used for debugging, so assume user is providing a good cipherstring *)
```

Comment:
I think there is another way to tell Cmdliner that an argument is required: this will immediately make the CLI fail on startup because the evaluation of failwith is not delayed.

Reply:
Cmdliner is opinionated about required parameters having to be unnamed positional ones, but it looks like it can be an option:

>`val required : 'a option t -> 'a Cmdliner.Term.t`

> `required` a is a term that fails if a's value is `None` and evaluates to the value of `Some` otherwise. Use this for required positional arguments (it can also be used for defining required optional arguments, but from a user interface perspective this shouldn't be done, it is a contradiction in terms).

Reply:
yes, you are right - thanks for pointing that out

i don't really want to make it `required` because as pau said i would then need to make it a positional argument, and this argument parser is a utility for other parsers, so that might be awkward - i will instead the `string` parser will something that fails if the cipherstring is empty